src/cow-react/modules/application/containers/App/Updaters.tsx

```diff
@@ -28,7 +27,6 @@ export function Updaters() {
       <ListsUpdater />
       <UserUpdater />
       <ApplicationUpdater />
-      <TransactionUpdater />
       <EnhancedTransactionUpdater />
```

Comment:
These 2 updaters do pretty much the same thing.
Seems like this caused a race condition where sometimes one would finalize the tx over the other.